src/v/cluster/partition_balancer_backend.cc

```diff
@@ -62,18 +62,14 @@ partition_balancer_backend::partition_balancer_backend(
   , _timer([this] { tick(); }) {}
 
 void partition_balancer_backend::start() {
-    if (is_enabled()) {
-        vlog(clusterlog.info, "partition autobalancing enabled");
-        _timer.arm(_tick_interval());
-    }
-
-    _mode.watch([this] { on_mode_changed(); });
+    _timer.arm(_tick_interval());
+    vlog(clusterlog.info, "partition balancer started");
 }
 
 void partition_balancer_backend::tick() {
     ssx::background = ssx::spawn_with_gate_then(_gate, [this] {
                           return do_tick().finally([this] {
-                              if (is_enabled() && !_gate.is_closed()) {
+                              if (!_gate.is_closed()) {
```

Comment:
nit: I think we should have a catch all and log from do_tick for debugging

Reply:
hmm, I'm probably misunderstanding the comment, but the catch-all is there (lines 78-79), no?

Reply:
you're right.. it was hidden in the github diff view, so I missed it. Resolving this comment.